lib/elements/rh-context-picker/rh-context-picker.ts

```diff
@@ -1,58 +1,113 @@
-import { type ColorPalette } from '../../context/color/provider.js';
 import { html, LitElement, type PropertyValues } from 'lit';
 import { customElement } from 'lit/decorators/custom-element.js';
 import { property } from 'lit/decorators/property.js';
 import { query } from 'lit/decorators/query.js';
 
+import { type ColorPalette } from '../../context/color/provider.js';
+import { colorContextConsumer, type ColorTheme } from '../../context/color/consumer.js';
+
+import type { Color } from '@rhds/tokens/js/types.js';
+import {
+  ColorSurfaceDarkest as darkest,
+  ColorSurfaceDarker as darker,
+  ColorSurfaceDark as dark,
+  ColorSurfaceLight as light,
+  ColorSurfaceLighter as lighter,
+  ColorSurfaceLightest as lightest,
+} from '@rhds/tokens/color.js';
```

Comment:
```Suggestion
} from '@rhds/tokens/js/color.js';
```

Reply:
Not in the export map it isn't

See tsconfig in this pr

Reply:
https://github.com/RedHat-UX/red-hat-design-system/blob/15dc5573d033cce047ec745aca542b0f3a8b55f1/tsconfig.json#L72-L74

Ok makes sense, didn't see the tsconfig change when working in #1298, copying the demo component over was complaining could find color.js.

Reply:
This is a hack the real solution is

module resolution: nodenext 

Also too lazy, mañana